### PR TITLE
Adding a Avro to AvroCopyable converter

### PR DESCRIPTION
**Pull request by sahilTakiar**

### CODE REVIEW

File: gobblin-core/src/main/java/gobblin/converter/avro/AvroToAvroCopyableConverter.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package gobblin.converter.avro;
```
Comment (liyinan926) [inline gobblin-core/src/main/java/gobblin/converter/avro/AvroToAvroCopyableConverter.java]: Missing License header.

**Comment by liyinan926** [comment]
LGTM other than the missing header.

**Comment by sahilTakiar** [comment]
Good catch! Just updated it.

**Comment by sahilTakiar** [comment]
Merged

**Pull request merged**